.env.example

```
@@ -0,0 +1,4 @@
+# Cognito settings (from Terraform outputs)
+VITE_COGNITO_REGION=ap-northeast-1
+VITE_COGNITO_USER_POOL_ID=
+VITE_COGNITO_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_COGNITO_CLIENT_ID key should go before the VITE_COGNITO_REGION key

(UnorderedKey)